agent/exploits/cve_2024_11205.py

```diff
@@ -0,0 +1,50 @@
+"""Agent Asteroid implementation for CVE-2024-11205"""
+
+import re
+
+from packaging import version
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VERSION_PATTERN = re.compile(r"Stable tag:\s(\d+\.\d+(?:\.\d+)*)")
+MAX_VULNERABLE_VERSION = version.parse("1.9.2.1")
+MIN_VULNERABLE_VERSION = version.parse("1.8.4")
+VULNERABILITY_TITLE = (
+    "WPForms Lite Plugin 1.8.4 - 1.9.2.1 - Missing Authorization to Authenticated "
+    "(Subscriber+) Payment Refund and Subscription Cancellation"
+)
+VULNERABILITY_REFERENCE = "CVE-2024-11205"
+VULNERABILITY_DESCRIPTION = (
+    "The WPForms Lite plugin for WordPress is vulnerable to unauthorized modification "
+    "of data due to a missing capability check on the 'wpforms_is_admin_page' function "
+    "in versions starting from 1.8.4 up to, and including, 1.9.2.1. This makes it "
+    "possible for authenticated attackers, with Subscriber-level access and above, to "
+    "refund payments and cancel subscriptions."
+)
+RISK_RATING = "HIGH"
+
+
+@exploits_registry.register
+class CVE202411205Exploit(webexploit.WebExploit):
+    accept_request = definitions.Request(
+        method="GET", path="/wp-content/plugins/wpforms-lite/readme.txt"
```

Comment:
We don't need other formats like "README.txt"

Reply:
![image](https://github.com/user-attachments/assets/2d9bdb08-ed56-4798-95fd-36fbea858d5d)